README.md

```diff
@@ -3,30 +3,30 @@ Install Ngninx and GH actions
 
 ## Install Ngninx and frameworks
 
-* SSH onto the VM
-* `git clone https://github.com/dltxio/dev-ops.git && cd dev-ops && ./install.sh`
-
-## Deploy bot SSH PubKey
-
-GitHub secrets add the following key pairs.  SSH private key can be found in bitwarden as "Digital Ocean Deploy Bot Private Key".
-
-* DO_DEPLOY_HOST
-* DO_DEPLOY_USERNAME
-* DO_DEPLOY_KEY
-
-DO_DEPLOY_HOST:  The IP or DNS of the VM  
-DO_DEPLOY_USERNAME:  The user name on the VM that will be used in the SSH tunnel  
-DO_DEPLOY_KEY: The SSH private key  
-
+1. Install the Digital Ocean Deploy Bot Private Key into your `~/.ssh` directory on your computer (found in BitWarden)
+2. SSH onto the VM
+3. `git clone https://github.com/dltxio/dev-ops.git && cd dev-ops && ./install.sh`
+
+## Deploy Bot SSH
+Allows GitHub actions in the CI yaml files to SSH onto your Digital Ocean server/droplet\
```

Comment:
Couple of typos
1. End of this line - why "\\"?
2. Line 12: "and add" -> "add"
3. Line 32 "Digital" is misspelled

Reply:
The ``\` at the end of the line is markdown for insert-new-line but without a new paragraph